src/test/common/page/page.js

```diff
@@ -133,6 +134,15 @@ class Page {
     const cookieLink = await this.findByCSS(COOKIES_LINK_CSS)
     await cookieLink.click()
   }
+
+  async getBackLink () {
+    try {
+      return this.findByClassName(BACK_LINK_CLASSNAME)
+    } catch (error) {
+      // throw error back up to test, needed for absence of link test.
```

Comment:
If you're intentionally doing this, what's the point in having the try/catch block at all if you just throw that error?

Reply:
This has been removed.

Reply:
This has been removed.